pinax/stripe/actions/customers.py

```diff
@@ -39,6 +39,20 @@ def create(user, card=None, plan=settings.PINAX_STRIPE_DEFAULT_PLAN, charge_imme
     Returns:
         the pinax.stripe.models.Customer object that was created
     """
+    try:
+        cus = models.Customer.objects.get(user=user)
+    except models.Customer.DoesNotExist:
+        pass
+    else:
+        try:
+            stripe.Customer.retrieve(cus.stripe_id)
+        except stripe.error.InvalidRequestError:
+            pass
+        else:
+            return cus
```

Comment:
I think it's better to just put this after the `stripe.Customer.retrieve` as `return cus` isn't going to raise an exception on its own and makes it more readable.

I also think something like this avoids an exception raise which again I think makes code a bit more readable.

```python
cus = models.Customer.objects.filter(user=user).first()
if cus is not None:
    try:
        stripe.Customer.retrieve(cus.stripe_id)
        return cus
    except stripe.error.InvalidRequestError:
        pass
```